packages/core/src/exposed-thing.ts

```diff
@@ -317,15 +317,20 @@ export default class ExposedThing extends TD.Thing implements WoT.ExposedThing {
         );
 
         if (this.properties[name]) {
-            let propertyHandler;
-            if (this.propertyHandlers.has(name)) {
-                propertyHandler = this.propertyHandlers.get(name);
-                propertyHandler.observeHandler = handler;
+            if (!this.properties[name].observable) {
+                throw new Error(
+                    `ExposedThing '${this.title}' cannot set observe handler for property '${name}' due to missing observable flag`
```

Comment:
Nit-picking: The observable flag cannot be "missing" since it is there by default.

"... due to conflicting observable flag"
OR
"since the observable flag is et to false"